recipes/msaprobs/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: msaprobs
+  version: 0.9.7
+
+source:
+  url: https://sourceforge.net/projects/msaprobs/files/MSAProbs-0.9.7.tar.gz
```

Comment:
... and then here write this as
```
url: https://sourceforge.net/projects/msaprobs/files/MSAProbs-{{ version }}.tar.gz
```
In that way, the version can be changed in a single position when a new release is mode.